awscli/argparser.py

```diff
@@ -108,6 +109,21 @@ def parse_known_args(self, args, namespace=None):
                 setattr(parsed, arg, encoded)
         return parsed, remaining
 
+    def error(self, message):
+        """error(message: string)
+
+        NOTE: This is lifted directly from argparse, the only difference being
+        we use 252 for parsing errors.
+
+        Prints a usage message incorporating the message to stderr and
+        exits.
+        If you override this in a subclass, it should not return -- it
+        should either exit or raise an exception.
+        """
+        self.print_usage(sys.stderr)
+        args = {'prog': self.prog, 'message': message}
+        self.exit(252, gettext('%(prog)s: error: %(message)s\n') % args)
```

Comment:
I think it may be worth while if we add a `constants` module and start defining all of these error codes, especially because we are using these in more than one module.

Reply:
Yeah, I was thinking about doing this. I'll break these out into constants.